src/test/java/seedu/guest/testutil/TypicalGuests.java

```diff
@@ -30,41 +30,47 @@ public class TypicalGuests {
             .withDateRange("30/09/22 - 02/10/22").withNumberOfGuests("2")
             .withPhone("98765432").withIsRoomClean("yes").withBill("1")
+            .withRequest("I want to clean my room for one more time")
             .build();
     public static final Guest CARL = new GuestBuilder().withName("Carl Kurz").withPhone("95352563")
             .withEmail("heinz@example.com").withRoom("05-25").withDateRange("30/12/22 - 02/01/23")
             .withNumberOfGuests("3").withIsRoomClean("no").withBill("19.99")
+            .withRequest("There are too many insects in my room")
             .build();
     public static final Guest DANIEL = new GuestBuilder().withName("Daniel Meier").withPhone("87652533")
             .withEmail("cornelia@example.com").withRoom("04-03").withDateRange("29/05/23 - 03/06/23")
             .withNumberOfGuests("4").withIsRoomClean("yes").withBill("50")
+            .withRequest("Nothing")
             .build();
     public static final Guest ELLE = new GuestBuilder().withName("Elle Meyer").withPhone("9482224")
             .withEmail("werner@example.com").withRoom("03-25").withDateRange("08/04/23 - 22/04/23")
-            .withNumberOfGuests("1").withIsRoomClean("no").withBill("100")
+            .withNumberOfGuests("1").withIsRoomClean("no").withBill("100").withRequest("Room service")
             .build();
     public static final Guest FIONA = new GuestBuilder().withName("Fiona Kunz").withPhone("9482427")
             .withEmail("lydia@example.com").withRoom("06-22").withDateRange("21/10/22 - 22/10/22")
             .withNumberOfGuests("2").withIsRoomClean("no").withBill("999.99")
+            .withRequest("The neighbour is too noisy, could you help me shoot him or her?")
             .build();
     public static final Guest GEORGE = new GuestBuilder().withName("George Best").withPhone("9482442")
             .withEmail("anna@example.com").withRoom("07-05").withDateRange("01/01/23 - 07/01/23")
             .withNumberOfGuests("3").withIsRoomClean("yes").withBill("1000.01")
+            .withRequest("Help me to fix Wi-Fi!")
             .build();
 
     // Manually added
     public static final Guest HOON = new GuestBuilder().withName("Hoon Meier").withPhone("8482424")
             .withEmail("stefan@example.com").withRoom("02-21").withDateRange("01/05/23 - 01/01/24")
-            .withNumberOfGuests("1").withIsRoomClean("no").withBill("6969.69")
+            .withNumberOfGuests("1").withIsRoomClean("no").withBill("6969.69").withRequest("Room service")
             .build();
     public static final Guest IDA = new GuestBuilder().withName("Ida Mueller").withPhone("8482131")
             .withEmail("hans@example.com").withRoom("05-36").withDateRange("11/11/23 - 12/11/23")
-            .withNumberOfGuests("4").withIsRoomClean("yes").withBill("9999999999")
+            .withNumberOfGuests("4").withIsRoomClean("yes").withBill("9999999999").withRequest("Room service")
```

Comment:
Should we have a different request compared to HOON?

Reply:
changed